src/WebApi/WebApi/UseCases/v1/Question/CreateQuestion/CreateQuestionRequest.cs

```diff
@@ -8,7 +9,8 @@ public record CreateQuestionRequest
     [MaxLength(100)]
     public string Title { get; set; }
 
-    [Range(1, 5)]
+    [Range(0, 100, ErrorMessage = "The Difficulty field is out of range ( 0 - 100 )")]
+    [Required(ErrorMessage = "The Difficulty field is required.")]
```

Comment:
`minor`

Since we don't seem to use the `ErrorMessage` anywhere else, I suppose we can skip it here. 🤔 

I kinda also think the default behavior generates basically this exact message. Not a 100% on that, though. ☺️ 

```suggestion
    [Required]
```